assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -1019,5 +1052,75 @@ describe( 'core/modules modules', () => {
 				expect( featuresLoaded ).toMatchObject( {} );
 			} );
 		} );
+
+		describe( 'hasModuleAccess', () => {
+			it( 'should use a resolver to make a network request', async () => {
+				fetchMock.postOnce(
+					/^\/google-site-kit\/v1\/core\/modules\/data\/check-access/,
+					{ body: { access: true }, status: 200 }
```

Comment:
You can omit the status property if it is `200` because this is the status by default:
```suggestion
					{ body: { access: true } }
```